### PR TITLE
feat: password dialogs

### PR DESCRIPTION
**Pull request by jhasuraj01**
- [x] Create Password Dialog
- [x] Remove Password Dialog
- [x] Change Password Dialog

### CODE REVIEW

**Comment by jhasuraj01** [comment]
Create Password Dialog:
Screenshot_20230908_200322

Remove Password Dialog:
Screenshot_20230908_200406

Change Password Dialog:
Screenshot_20230908_200425

File: packages/cysync-core/src/components/GenericLoader.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <DialogBox width={500} height={300}>
+    <Image
+      src={loaderGrayIcon}
+      width={68}
+      alt="Loader icon"
+      animate="spin"
+      $animDuration={3}
+    />
+  </DialogBox>
+);
```
Comment (irshadCypherock) [inline packages/cysync-core/src/components/GenericLoader.tsx]: This already exists as `LoaderDialog` in the same folder

File: packages/cysync-core/src/constants/i18n/en.ts [hunk 2]
```diff
@@ -1276,6 +1276,38 @@ const en = {
...
+          title: 'Confirm password to proceed',
+          subTitle: 'Provide password to proceed',
+          info: 'Use 8 or more characters with a mix of letters, numbers & symbols',
+        },
+      },
+      setPassword: {
+        createNewPassword: {
+          title: 'Set your cySync password',
+          subTitle: 'We do not store your password on our servers',
+          info: 'Use 8 or more characters with a mix of letters, numbers & symbols',
```
Comment (irshadCypherock) [inline packages/cysync-core/src/constants/i18n/en.ts]: Repeated text

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: packages/cysync-core/src/components/GenericLoader.tsx (hunk 1 above)
Comment (jhasuraj01) [reply packages/cysync-core/src/components/GenericLoader.tsx]: deleted unwanted file

File: packages/cysync-core/src/constants/i18n/en.ts (hunk 2 above)
Comment (jhasuraj01) [reply packages/cysync-core/src/constants/i18n/en.ts]: fix: reduced redundancy

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**